static/app/components/group/suggestedOwners/ownershipRules.tsx

```diff
@@ -78,3 +144,51 @@ const HelpfulBody = styled('div')`
   padding: ${space(1)};
   text-align: center;
 `;
+
+const Container = styled(Panel)`
+  width: 100%;
+  height: 100%;
+  background: none;
```

Comment:
Is this also needed?

Reply:
@EvanPurkhiser need the `background:none` to keep the background of the prompt white

Reply:
what color does it become otherwise?

Reply:
`'#FAF9FB'` backgroundSecondary

Reply:
ahh yeah icic